posthog/api/test/test_capture.py

```diff
@@ -155,6 +155,24 @@ def test_emojis_in_text(self, patch_process_event_with_plugins):
             "💻 Writing code",
         )
 
+    @patch("posthog.models.team.TEAM_CACHE", {})
+    @patch("posthog.api.capture.celery_app.send_task")
+    def test_js_gzip(self, patch_process_event_with_plugins):
```

Comment:
Maybe worth adding a test for an incorrectly formatted payload with `gzip-js` to make sure it's handled gracefully?

Reply:
Good question. Unsure what the expected behavior here is - silently eating errors in the util would lead to problems elsewhere.

Reply:
I'm making a proposal change for this, will just make sure a proper error response is returned instead of a cryptic 500

Reply:
Never mind, as the `/capture` endpoint is not on DRF, handling the exception requires a more comprehensive refactor, tabling this for now into a separate issue.